androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/NpsBar.kt

```diff
@@ -0,0 +1,112 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyAnswerUiModel
+
+@Composable
+fun NpsBar(
+    surveyAnswerUiModels: List<SurveyAnswerUiModel>,
+    modifier: Modifier = Modifier,
+) {
+    var selectedIndex by remember { mutableStateOf(0) }
+    Box(
+        contentAlignment = Alignment.TopCenter,
+        modifier = modifier
+            .wrapContentWidth()
+            .height(94.dp)
+    ) {
+        LazyRow(
+            modifier = Modifier
+                .border(BorderStroke(1.dp, White), MaterialTheme.shapes.medium)
+                .height(57.dp)
+        ) {
+            items(surveyAnswerUiModels.size) { index ->
```

Comment:
```suggestion
            items(
                count = surveyAnswerUiModels.size,
                key = { surveyAnswerUiModels[it].id },
            ) { index ->
```

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/126/commits/32e874347181ddd2ff1acf387ad8cbd887b1034e